sdlib/d/gc/capi.d

```diff
@@ -7,6 +7,7 @@ extern(C):
 /**
  * Standard C allocating functions.
  */
+/+
```

Comment:
I don't understand why this is here. This is already commented in sdc's branch. Is github doing something retarded?

Reply:
I can only say, I think so. I agree it does look like this is already there.